src/utils/FirebaseLib.js

```diff
@@ -27,6 +27,15 @@ function fireBaseWrapper(){
   this.dataOn = function(eventType, callBack){
     this.fireBase.on(eventType, callBack);
   };
+
+  this.getEloDateForCurrentMonth = function(playerId, eventType, callBack){
```

Comment:
` this.getEloDateForCurrentMonth`

Should this be 

` this.getEloDataForCurrentMonth`